src/main/java/sirius/biz/tycho/kb/KnowledgeBaseController.java

```diff
@@ -116,4 +119,21 @@ private boolean checkArticlePermission(KnowledgeBaseArticle article, String auth
                                    authKey)
                || Strings.areEqual(article.computeAuthenticationSignature(false), authKey);
     }
+
+    /**
+     * Provides an autocompletion for KB articles.
+     *
+     * @param webContext the current request
+     */
+    @Permission(TenantUserManager.PERMISSION_SYSTEM_TENANT_MEMBER)
+    @Routed(value = "/kb/autocomplete", priority = 99)
+    public void tenantsAutocomplete(final WebContext webContext) {
+        AutocompleteHelper.handle(webContext, (query, result) -> {
+            knowledgeBase.query(null, query, 25).forEach(article -> {
```

Comment:
Optional: Define a `AutocompleteHelper#DEFAULT_LIMIT` constant (also note OX uses a limit of 10 in most places, maybe we should unify that)

Reply:
https://github.com/scireum/sirius-web/pull/1092